corehq/apps/hqadmin/templates/hqadmin/bootstrap3/hqadmin_base_filters.html

```diff
@@ -0,0 +1,36 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div class="hq-accordion-control">
```

Comment:
Confused by this - B3 doesn't have the `.accordion-foo` classes anymore, is this an hq-specific widget?